### ISSUE TITLE
`warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint` on m1 Mac even though SwiftLint is installed through Homebrew

### ISSUE THREAD

**Issue by agisilaos** [realm/SwiftLint#3777]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

Hello there. I installed SwiftLint through Homebrew on my new m1 machine. As I'm trying to run the Xcode Project I get a warning that `SwiftLint not installed, download from https://github.com/realm/SwiftLint`SwiftLint is indeed installed though. 

 I know that I can modify the script in Xcode to 

```bash
if test -d "/opt/homebrew/bin/"; then
  PATH="/opt/homebrew/bin/:${PATH}"
fi

export PATH

if which swiftlint >/dev/null; then
  swiftlint
else
  echo "warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint"
fi
```

But I want to fix it locally without the need of changing the script from within Xcode. Couldn't find any particular useful info on the topic. Can anyone point me to the right direction? How I need to modify my `.zshrc` file?

Thank you all in advance.

**Comment by agisilaos** [comment]
Planning to close this issue because after discussing it internally the way forward is changing the script. Planning to make a PR and updating the docs.

**Comment by b593771943** [comment]
good question, 3Q for your script, expecting author can be attention on this problem, provide the final solution which just modify my .zshrc file or .bash_profile file
